### PR TITLE
Added more links to awesome rust projects for the nRF52 series

### PR DESCRIPTION
**Pull request by sigvartmh**

### CODE REVIEW

**Comment by jamesmunns** [comment]
Hey @sigvartmh, thanks for the contribution!

As one of the maintainers of `nrf-rs`, it would be nice to link to the `https://github.com/nrf-rs` organization directly, similar to how `stm32-rs` is grouped.

Let me know what you think :)

**Comment by sigvartmh** [comment]
Yup that would be better @jamesmunns awesome stuff maintainig `nrf-rs` btw keep up the cool work.

**Comment by jamesmunns** [comment]
Most of the credit goes to @wez and @hannobraun :)

**Comment by RandomInsano** [comment]
Repo link is dead [on the nrf52840 crate](https://crates.io/crates/nrf52840), but maybe that's an @wez issue here?

Similar for [the nrf52-hal crate](https://crates.io/crates/nrf52-hal) but that seems to be that it links to an archived Repo from before the move.

Is this just a limitation of Crates.io I wasn't aware of?

**Comment by sigvartmh** [comment]
Changed the structure a bit @RandomInsano also added nrf52dk-sys crate if that's okei for @jamesmunns

**Comment by wez** [comment]
We're in between transitioning from some old crate and repo names and the new ones in the new org, so we may have some old names mentioned, some new names not yet published, and some stuff may be slightly wrong at the moment.  We'll get our ducks in a row and ping this issue when its a bit saner.

File: README.md [hunk 1]
```diff
@@ -217,6 +219,9 @@ Crates tailored for specific boards.
 
 [micro:bit]: http://microbit.org/
 
+- ['nrf52832-dk'](https://crates.io/crates/nrf52dk-sys) - Repo for the nRF52832-dk with bindings to the S132 softdevice for bluetooth connectivity
```
Comment (therealprof) [inline README.md]: Wrong quotes used here.

File: README.md [hunk 2]
```diff
@@ -98,6 +98,7 @@ Register definition for microcontroller families. Usually generated using [`svd2
 ### Nordic
 
 - [`nrf51`](https://crates.io/crates/nrf51) Peripheral access API for nRF51 microcontrollers (generated using svd2rust) - ![crates.io](https://img.shields.io/crates/v/nrf51.svg)
+- [`nrf52-rs`](https://github.com/nrf-rs) Peripheral access API for nRF52 series microncontrollers (generated using svd2rust): [`nrf52840`](https://crates.io/crates/nrf52840) [`nrf52832`](https://crates.io/crates/nrf52832)
```
Comment (therealprof) [inline README.md]: https://crates.io/crates/nrf52832 does not exist

Comment (sigvartmh) [inline README.md]:
```suggestion
[`nrf52-rs`](https://github.com/nrf-rs) Peripheral access API for nRF52 series microncontrollers (generated using svd2rust): [`nrf52840`](https://crates.io/crates/nrf52840) [`nrf52832`](https://crates.io/crates/nrf52832-hal)
```

File: README.md (hunk 1 above)
Comment (sigvartmh) [inline README.md]:
```suggestion
[`nrf52832-dk`](https://crates.io/crates/nrf52dk-sys) - Repo for the nRF52832-dk with bindings to the S132 softdevice for bluetooth connectivity

**Comment by sigvartmh** [comment]
@therealprof They are doing major rework on their repository structure and naming so maybe wait with merge until they've finished https://crates.io/keywords/nrf52

**Comment by therealprof** [comment]
@sigvartmh I'm kind of aware since I'm part of that organisation. Would it be okay for you to close this in the meanwhile until the dust has settled?

**Comment by sigvartmh** [comment]
Will do 👍

**Pull request closed without merging**